docs/extensions/capabilities/common-capabilities.md

```diff
@@ -0,0 +1,258 @@
+# Common Capabilities
+
+Common Capabilities are important building blocks for your extensions. Almost all extensions use some of these functionalities. Here is how you can take advantage of them.
+
+## Main Extension
+
+A main extension runs in the background and it's not visible directly via Lens main window. If you want to see logs from this extension you need to start Lens from the command line.
```

Comment:
Do you mean a main extension object does not add content directly in the UI? Maybe something like:
A main extension runs in the background and, apart from app menu items, does not add content to the Lens UI.

Reply:
Yes, thanks!